exercises/bob/examples/success-standard/src/Bob.hs

```diff
@@ -1,5 +1,6 @@
 module Bob (responseFor) where
-import Data.Char (isSpace, isUpper, isAlpha)
+import Data.Char (isSpace, isUpper, isAlpha, isPunctuation)
+import Safe
```

Comment:
since only `lastMay` is being used from `Safe`, I'd suggest making that clear by listing it. Otherwise it may not be clear when/if it is safe to remove `import Safe` if code ever changes in the future.